app/src/org/commcare/AppUtils.java

```diff
@@ -177,6 +177,13 @@ public static String getCurrentVersionString() {
         return Localization.get(application.getString(R.string.app_version_string), new String[]{pi.versionName, String.valueOf(pi.versionCode), ccv, buildNumber, buildDate, profileVersion});
     }
 
+    public static String getCommCareVersion() throws PackageManager.NameNotFoundException {
+        CommCareApplication application = CommCareApplication.instance();
+        PackageManager pm = application.getPackageManager();
+        PackageInfo pi = pm.getPackageInfo(application.getPackageName(), 0);
+        return pi.versionName;
```

Comment:
Curious how is it different from `BuildConfig.VERSION_NAME`

Reply:
ahh I should definitely be using `BuildConfig.VERSION_NAME` here. thanks for flagging.